Backend/ManagerLayer/Login/UserLoginManager.cs

```diff
@@ -18,9 +19,10 @@ public class UserLoginManager
         UserManagementManager _userManagementManager;
         AuthorizationManager _authorizationManager;
         TokenService _tokenService;
+        LogRequestDTO newLog;
 
         public LoginManagerResponseDTO LoginFromSSO(
-            DatabaseContext _db, string Username, Guid ssoID, string Signature, string PreSignatureString)
+            DatabaseContext _db, string Username, Guid ssoID, string Signature, string Timestamp, string PreSignatureString)

```

Comment:
We gonna change the in parameters because of logs? Cant generate a timestamp within this scope?

Reply:
Timestamp generated will not match the timestamp used for the signature

Reply:
Make sure that change is reflected everywhere